api/RTCIceTransport.json

```diff
@@ -250,13 +250,13 @@
           "description": "<code>gatheringstatechange</code> event",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "75"
```

Comment:
I can confirm this, it was implemented in https://source.chromium.org/chromium/chromium/src/+/d78d9f90c87a37772bbc55a72fb085d2c86faa85 before the whole feature was enabled, and thus shipped with the rest of `RTCIceTransport`.